docs/en/seeding-classic-image.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Seeding a Classic Image"
+table_of_contents: true
+---
+
+# Seeding a Classic Image
+
+## Introduction
+A Classic image is seeded to include snaps and [assertions][assertions]. That
+is, you add the snaps and the assertions to the image so that on first boot it
+contains what your product needs. For example, a [gadget snap][gadget-snap] is
+normally seeded to point the system at a Serial Vault. Also, one seeds snaps
+to provide the software that customizes the system. This document explains
+what you may want to seed and how to do it.
+
+
+!!! Note:
+           Core images are seeded automatically with snaps and the model
+          assertion when created with the normal  ubuntu-image tool (available
+          as a snap in the beta channel). But, ubuntu-image does not yet
+          support building Classic images. However, the principles explained
+          here can be used to manually modify a Core image after it is built
+to
+          seed additional snaps and assertions.
+
+## What to seed
+Usually one needs to seed:
+
+ - A model assertion.
+ - An account assertion: for the brand SSO account used to authenticate the
+model assertion.
+ - An account-key assertion: to authenticate the key that signs the model
+assertion.
+ - A gadget snap and its assertions.
+
+!!! Note:
+          The gadget snap uploader must be the brand SSO account
```

Comment:
I think this is slightly confusing. It would be better to say, "The gadget snap must be uploaded using the brand SSO account"